docs/source/start.ipynb

```diff
@@ -37,7 +38,32 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "X, y = evalml.demos.load_breast_cancer()"
+    "X, y = evalml.demos.load_fraud(n_rows=1000)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "First, we want to clean the data. Since EvalML accepts pandas input, it will run type inference on top of the input pandas data. If you\u2019d like to change the types inferred by EvalML, you can use the `infer_feature_types` utility method as follows. The `infer_feature_types` utility method takes pandas or numpy input and converts it to a [Woodwork](https://woodwork.alteryx.com/en/stable/) data structure. It takes in a `feature_types` parameter which can be used to specify what types specific columns should be. In the example below, we specify the types that we want for certain columns, such as the provider column. The column 'provider' would have been inferred as a column with natural language, but we want to specify it as a categorical column instead. "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# convert X to pandas dataframe in order for us to change the data\n",
+    "X = X.to_dataframe()\n",
+    "X['expiration_date'] = X['expiration_date'].apply(lambda x: '20{}-01-{}'.format(x.split(\"/\")[1], x.split(\"/\")[0]))\n",
+    "X[['lat', 'lng']] = X[['lat', 'lng']].astype('str')\n",
+    "X = infer_feature_types(X, feature_types= {'store_id': 'categorical',\n",
```

Comment:
I see lat, lng being manipulated here but not mentioned at all in the markdown. Do we need this change?

Reply:
@bchen1116 
You could perhaps provide a brief list just describing what you're doing here:

First, let's clean the data. Since EvalML accepts a pandas input, it can run type inference on this data directly. If you’d like to change the types inferred by EvalML, you can use the infer_feature_types utility method. Here's what we're going to do:
- Reformat the `expiration_date` column so it reflects a more familiar date format.
- Cast the `lat` and `lng` columns from `float` to `str`.
- Use `infer_feature_types` to specify what types certain columns should be. For example, to avoid having the column `provider` be inferred as one based on text, we have specified it as `categorical` instead.

The infer_feature_types utility method takes a pandas or numpy input and converts it to a Woodwork data structure, providing you with flexibility!